oshi-core/src/main/java/oshi/software/os/linux/LinuxOSProcess.java

```diff
@@ -119,6 +158,16 @@ public String getCommandLine() {
         return this.commandLine;
     }
 
+    @Override
+    public List<String> getArguments() {
+        return this.arguments;
```

Comment:
OK now I see why you made the parsing methods return unmodifiable, you directly return them here.  I think there's a better way to do this.  I'm wondering if a memoizer would be a better implementation for these (as well as the command line).